microfragments/src/main/java/org/dmfs/android/microfragments/transitions/BackTransition.java

```diff
@@ -110,7 +110,15 @@ public Timestamp timestamp()
     @Override
     public void prepare(@NonNull Context context, @NonNull FragmentManager fragmentManager, @NonNull MicroFragmentHost host, @NonNull MicroFragment<?> previousStep)
     {
-        mResponseCage.pigeon(fragmentManager.popBackStackImmediate()).send(context);
+        boolean result = fragmentManager.popBackStackImmediate();
```

Comment:
No, first we pop the current fragment but only if the backstack is non-empty.
If the next one is a "skipable" MFs (i.e a loader) we pop it an all other skipables right in front of it.

The backstack can't contain only skipables, because skipables are not added to the backstack if it's empty (because there is no need to).